registration-processor/post-processor/registration-processor-printing-stage/src/main/java/io/mosip/registration/processor/print/stage/PrintingStage.java

```diff
@@ -354,6 +353,9 @@ private CredentialRequestDto getCredentialRequestDto(String regId) {
 		credentialRequestDto.setIssuer(env.getProperty("mosip.registration.processor.issuer"));
 
 		credentialRequestDto.setEncryptionKey(generatePin());
+		additionalAttributes.put("credential_id",regId);
```

Comment:
I believe we are calling this method passing the VID, do we want to set the vid as credential id ?

Reply:
also the AID#PDF convention is not followed

Reply:
Also I believe credential id should in credentialRequestDto, not as additional attributes, this has to be /v2 API

Reply:
taken care